emcc.py

```diff
@@ -1405,6 +1406,11 @@ def consume_arg_file():
       settings.BULK_MEMORY = 1
     elif arg == '-mno-bulk-memory':
       settings.BULK_MEMORY = 0
+    elif arg == '-msign-ext':
+      settings.SIGN_EXT = 1
+      feature_matrix.enable_feature(feature_matrix.Feature.SIGN_EXT, '-msign-ext')
+    elif arg == '-mno-sign-ext':
+      feature_matrix.disable_feature(feature_matrix.Feature.SIGN_EXT)
```

Comment:
Why not do the same for bulk memory?  I guess because we don't have a link time lowering yet?

Reply:
Right, link time lowering is in my local branch but not upstream yet.

Reply:
OK, perhaps we can generalize this once we do the same for bulk memory, perhaps even make it data driven?  e.g.  

```
# map command line feature flags to (setting, feature) pairs 
feature_flags = {
  'bulk-memory': ('BULK_MEMORY', feature_matrix.Feature.SIGN_EXT),
  ...
}
```

Reply:
Yeah, I'll come back to the best way to do this this with bulk and nontrapping.